frontend/package.json

```diff
@@ -3,17 +3,28 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "bulma": "^0.7.2",
+    "bulma-extensions": "^4.0.1",
     "classnames": "^2.2.6",
+    "mediaquery": "0.0.4",
+    "node-sass-chokidar": "^1.3.4",
```

Comment:
Now that I think about it, that should probably be replaced with `node-sass` (following https://facebook.github.io/create-react-app/docs/adding-a-sass-stylesheet), and everything will be handled by create-react-app